src/message.ts

```diff
@@ -1,5 +1,5 @@
-interface MessageHeaders {
-  recipients: string;
+type MessageHeaders = {
```

Comment:
This is kind of holy-war change, why you did it?

Reply:
I bought back interface here.